dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/ValidationExecution.java

```diff
@@ -37,23 +37,26 @@ public class ValidationExecution extends Execution {
     @NotNull
     @JsonProperty(required = true)
     @Schema(description = "The validator tool used to validate the workflow", required = true, example = "miniwdl")
-    ValidatorTool validatorTool;
+    private ValidatorTool validatorTool;
 
     @NotNull
     @JsonProperty(required = true)
     @Schema(description = "The version of the validator tool", required = true)
-    String validatorToolVersion;
+    private String validatorToolVersion;
 
     @NotNull
     @JsonProperty(required = true)
     @Schema(description = "Boolean indicating if the workflow was validated successfully", required = true, example = "true")
-    Boolean isValid;
+    private Boolean isValid;
+
+    @Schema(description = "The error message for a failed validation by the validator tool")
+    private String errorMessage;
```

Comment:
I could envision this potentially being an array of error messages, like a compiler or linter might produce. But we're only getting true/false submitted for now, so we can cross this bridge later if it comes up; no need to change.